<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>650</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>650</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-11</y>
         <width>591</width>
         <height>431</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>-40</x>
         <y>0</y>
         <width>1241</width>
         <height>641</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/new/prefix1/dashboard.jpg)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="noviArtikal">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>560</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(38, 255, 23);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>DODAJ ARTIKAL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>560</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#03a9f4;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>UCITAJ ARTIKLE</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>521</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="dobrodosli">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>0</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font:15px;
background-color:green;
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="stanje">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
color:white;
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Broj dostupnih bicikala:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="stanjeBAM">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
color:white;
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Kolicina bicikala (BAM):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="noviArtikal_2">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>490</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(18, 25, 23);
border-radius: 15px;
color:#fff;</string>
       </property>
       <property name="text">
        <string>HISTORIJA TRANSAKCIJA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>560</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:red;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>PRODAJ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(18, 25, 23);
border-radius: 15px;
color:#fff;</string>
       </property>
       <property name="text">
        <string>ODJAVA</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>1201</width>
         <height>661</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>-30</x>
          <y>-20</y>
          <width>1251</width>
          <height>681</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/new/prefix1/dodajartikal.jpg)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>570</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(38, 255, 23);
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string>POCETNA</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>480</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(38, 255, 23);
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string>DODAJ</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font:15px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;DODAJ NOVI ARTIKAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>541</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineIme">
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:9px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineCijena">
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:9px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineRam">
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:9px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineBrzine">
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:9px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineStanje">
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:9px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1301</width>
         <height>771</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>-220</x>
          <y>-30</y>
          <width>1441</width>
          <height>681</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/new/prefix1/RD53383-scaled.jpg)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>1010</x>
          <y>570</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(38, 255, 23);
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string>POCETNA</string>
        </property>
       </widget>
       <widget class="QLabel" name="stanje_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:red;
color:white;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Historija transakcija&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="ucitajTransakcije">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>560</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:#03a9f4;
border-radius:15px;</string>
        </property>
        <property name="text">
         <string>UCITAJ TRANSAKCIJE</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>361</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="stanjeArtikalaBAM">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:red;
color:white;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
